### PR TITLE
implemented dashboard structure

### PR DESCRIPTION
**Pull request by BilalMaqsoodA**

### CODE REVIEW

File: src/components/ui/status/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Text } from '../text';
+
+import { Wrapper } from './styled';
+
+import { Badge } from './badge';
+
+export const Status = () => (
+  <Wrapper>
```
Comment (InamTaj) [inline src/components/ui/status/index.js]: Please be consistent in naming, this should either by `Styled` or `StyledWrapper`

File: src/components/ui/status/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Text } from '../text';
+
+import { Wrapper } from './styled';
+
+import { Badge } from './badge';
```
Comment (InamTaj) [inline src/components/ui/status/index.js]: Move this import above with Text import

File: src/components/ui/status/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (InamTaj) [inline src/components/ui/status/index.js]: Please see existing components, the index  only exports the UI component, in this case, UI component will be `status.js` which will be exported from this `index.js` entry point

File: src/components/ui/status/badge/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const Badge = styled.div`
+  height: 20px;
+  width: 20px;
+  background-color: red ;
+  border-radius: 50%;
+  display: inline-block;
+
```
Comment (InamTaj) [inline src/components/ui/status/badge/index.js]: Why empty line?

File: src/components/ui/status/badge/index.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+
+export const Badge = styled.div`
+  height: 20px;
+  width: 20px;
+  background-color: red ;
+  border-radius: 50%;
+  display: inline-block;
```
Comment (InamTaj) [inline src/components/ui/status/badge/index.js]: Sort all CSS properties alphabetically. Did you read the CONTRIBUTING Guide?

File: src/components/ui/status/badge/index.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
```
Comment (InamTaj) [inline src/components/ui/status/badge/index.js]: Please see existing components, the index  only exports the UI component, in this case, UI component will be `badge.js` which will be exported from this `index.js` entry point

File: src/components/ui/button/index.js [hunk 7]
```diff
@@ -0,0 +1,4 @@
+import styled from 'styled-components';
+import {Button as rButton} from 'rebass';
```
Comment (InamTaj) [inline src/components/ui/button/index.js]: Give spaces inside brackets

File: src/components/ui/button/index.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+import styled from 'styled-components';
```
Comment (InamTaj) [inline src/components/ui/button/index.js]: Please see existing components, the index  only exports the UI component, in this case, UI component will be `button.js` which will be exported from this `index.js` entry point

File: src/components/header/index.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+} from '../ui/nav';
+
+import {
+  StyledWrapper,
+} from './styled';
+
+
+export const Header = () => (
+  <StyledWrapper>
+    <Nav
```
Comment (InamTaj) [inline src/components/header/index.js]: Bit of misunderstanding here, the `Nav` isn't just a wrapper on Link, it is a whole Navigation bar which includes all links, please fix accordingly.

File: src/components/dashboard/orders/order-search/index.js [hunk 10]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (InamTaj) [inline src/components/dashboard/orders/order-search/index.js]: Please see existing components, the index  only exports the UI component, in this case, UI component will be `order-search.js` which will be exported from this `index.js` entry point

File: src/components/dashboard/orders/order-detail/index.js [hunk 11]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (InamTaj) [inline src/components/dashboard/orders/order-detail/index.js]: Please see existing components, the index  only exports the UI component, in this case, UI component will be `order-detail.js` which will be exported from this `index.js` entry point

File: src/components/dashboard/orders/index.js [hunk 12]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (InamTaj) [inline src/components/dashboard/orders/index.js]: Please see existing components, the index  only exports the UI component, in this case, UI component will be `orders.js` which will be exported from this `index.js` entry point

File: src/components/header/index.js (hunk 9 above)
Comment (BilalMaqsoodA) [reply src/components/header/index.js]: and also link component separately ?

File: src/components/dashboard/orders/index.js (hunk 12 above)
Comment (InamTaj) [reply src/components/dashboard/orders/index.js]: Still pending

File: src/components/ui/status/status.js [hunk 13]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Text } from '../text';
+import { Badge } from './badge';
+import { Styled} from './styled';
```
Comment (InamTaj) [inline src/components/ui/status/status.js]: add a space before closing bracket

**Pull request merged**